layers/meta-resin-edison/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -0,0 +1,10 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/${PN}"
+
+SRC_URI_append = " \
```

Comment:
You can clean this up a little. All you need is a prepend in FILESEXTRAPATHS and in that path have the file that we copy in the main meta-resin recipe. In this way you don't need any SRC_URI or do_install code.